aiogram/filters/command.py

```diff
@@ -78,6 +78,17 @@ def __init__(
                 " or their Iterable"
             )
 
+        prefixes = []
```

Comment:
Why do you need to separate prefixes into list of chars?

`for symbol in "abc"` is the same as `for symbol in ["a", "b", "c"]`, so, you don't need to rebuild it in this way.

Also when you need to convert string into list of chars, recommended to use list("abc") instead of for-loop,
also if chars should be unique you can use set instead of list.